functional-test/pages/tabs/appealDetails.ts

```diff
@@ -29,8 +31,15 @@ export class AppealDetails {
     async verifydueDates(reqField: string){
         const dueDate = new Date();
         dueDate.setDate(new Date().getDate());
-        let fomattedDueDate = dateUtilsComponent.formatDateToSpecifiedDateShortFormat(dueDate);
-        await this.verifyAppealDetailsPageContentByKeyValue(reqField, fomattedDueDate);
+        let formattedDueDate = dateUtilsComponent.formatDateToSpecifiedDateShortFormat(dueDate);
+
+        //Java has replaced the short of September for 'en-GB' locale to be 'Sept' which is failing our tests, this regex is a workaround for that
+        if(currentMonth === 8){
+            formattedDueDate = formattedDueDate.replace(/\bSept\b/, "Sep");
```

Comment:
Ideally we should come up with a solution in the Utils method.we will investigate further.